### PR TITLE
Implement 'find' feature to search for matching property

### LINKED ISSUE #106: As a property manager, I can search for matching text so that i can reduce my search to a smaller scope.

### PR DESCRIPTION
**Pull request by wilsonngja**
Fix #106

### CODE REVIEW

**Review by OVReader — COMMENTED** [review]
Overall LGTM, coding standards are adhered.

File: src/main/java/seedu/duke/parsermanager/ParseFindProperty.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        this.commandDescription = commandDescription;
+    }
+
+    @Override
+    public Command parseCommand() throws DukeException {
+        boolean hasNoDescription = commandDescription.trim().isEmpty();
+        boolean isValidDescription = checkCommandValidity(commandDescription);
+
+        if (hasNoDescription) {
+            throw new FindDescriptionEmptyException(MESSAGE_EMPTY_DESCRIPTION);
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/parsermanager/ParseFindProperty.java]: Please use the standardised exceptions

**Review by ngdeqi — COMMENTED** [review]
Do use the standardised messages and exceptions

**Review by FeliciaBeatrice — APPROVED** [review]
everything looks good! LGTM

**Pull request merged**